easybuild/tools/systemtools.py

```diff
@@ -49,11 +49,24 @@
 _log = fancylogger.getLogger('systemtools', fname=False)
 
 # constants
+AARCH32 = 'AArch32'
+AARCH64 = 'AArch64'
+POWER = 'POWER'
+X86_64 = 'x86_64'
+
 AMD = 'AMD'
```

Comment:
make it clear that these are vendors, while the above are architectures (with a comment)